extension/observer/dockerobserver/extension.go

```diff
@@ -69,14 +84,37 @@ func (d *dockerObserver) Start(ctx context.Context, host component.Host) error {
 		return fmt.Errorf("could not create docker client: %w", err)
 	}
 
-	// Load initial set of containers
-	err = d.dClient.LoadContainerList(d.ctx)
+	err = d.syncContainerList()
```

Comment:
by previously not emitting container endpoints the observer wouldn't report any container running at collector startup.